frontend/src/app/home/home.component.ts

```diff
@@ -10,8 +12,10 @@ import { Flashcard } from '../_models/flashcard.class';
 })
 export class HomeComponent {
   subjectId = 0;
+  tagControl = new FormControl();
   private cardIndex = 0;
   private subjectMap: { [key: number]: Subject } = {};
+  private tagMap: { [key: number]: Tag } = {};
```

Comment:
We never use the implemented map functionality, do we? A simple array should be sufficient...

Reply:
Yes, that is right.